Source/WebCore/Modules/webcodecs/WebCodecsAudioDecoder.cpp

```diff
@@ -63,9 +63,27 @@ WebCodecsAudioDecoder::~WebCodecsAudioDecoder()
 {
 }
 
-static bool isValidDecoderConfig(const WebCodecsAudioDecoderConfig& config, const Settings::Values&)
+static bool isValidDecoderConfig(const WebCodecsAudioDecoderConfig& config)
 {
-    return AudioDecoder::isCodecSupported(config.codec);
+    if (StringView(config.codec).trim(isASCIIWhitespace<UChar>).isEmpty())
+        return false;
+
+    if (!config.numberOfChannels)
+        return false;
+
+    if (!*config.numberOfChannels)
+        return false;
+
+    if (!config.sampleRate)
+        return false;
+
+    if (!*config.sampleRate)
+        return false;
+
+    if ((config.codec == "flac"_s || config.codec == "vorbis"_s ) && !config.description)
+        return false;
```

Comment:
I do not see this check in the spec. I would think configure would close the decoder instead.

Reply:
That's mandated by the flac and vorbis codec registries. Blink has similar code, but I can remove it. AFAICS there's no WPT test checking this.

Reply:
Removed.